apps/src/codebridge/FileBrowser/FileBrowserRow/hooks/useStartModeFileRowOptions.ts

```diff
@@ -0,0 +1,106 @@
+import {useCodebridgeContext} from '@codebridge/codebridgeContext';
+import {ProjectFile} from '@codebridge/types';
+import {useMemo} from 'react';
+
+import codebridgeI18n from '@cdo/apps/codebridge/locale';
+import {START_SOURCES} from '@cdo/apps/lab2/constants';
+import {setOverrideValidations} from '@cdo/apps/lab2/lab2Redux';
+import {PASSED_ALL_TESTS_VALIDATION} from '@cdo/apps/lab2/progress/constants';
+import {getAppOptionsEditBlocks} from '@cdo/apps/lab2/projects/utils';
+import {ProjectFileType} from '@cdo/apps/lab2/types';
+import {useAppDispatch} from '@cdo/apps/util/reduxHooks';
+
+import {setShowLockedFilesBanner} from '../../../redux/workspaceRedux';
+
+/**
+ * Dropdown options for the file dropdown in start mode.
```

Comment:
Can you update the comment to explain this will return an empty list if we are not in start mode?